### PR TITLE
Add Terraform convenience features

### PR DESCRIPTION
**Pull request by bellackn**

### CODE REVIEW

**Comment by tobias-richter** [comment]
@bellackn please review the travis build.

**Comment by bellackn** [comment]
> 
> 
> @bellackn please review the travis build.

Oops, just an indentation issue. Fixed now.

File: README.md [hunk 1]
```diff
@@ -221,6 +221,19 @@ variable definition in the vars folder.
 
 Enables / disables os package installation.
 
+    controlhost_terraform_utilities
+    
+Enables / disables Terraform convenience utilities setup, including Bash aliases and setting `TF_DATA_DIR`.
+This also allows for using a non-rsync folder sync with Windows hosts due to Terraform symlinks being created outside of
+the sync scope.
+
+    # controlhost_terraform_project_name: sample-project
```
Comment (tobias-richter) [inline README.md]: we should name this parameter controlhost_terraform_data_dir. We can then set this in the archetype to the project name. But the controlhost does not know or referenc to any project.

File: defaults/main.yml [hunk 2]
```diff
@@ -123,3 +123,9 @@ controlhost_packages_pip:
 
 # enables / disables os package installation
 controlhost_packages_os_enabled: true
+
+# Enables / disables Terraform convenience utilities setup
+controlhost_terraform_utilities: true
+
+# Set a Terraform project name that will be used in TF_DATA_DIR variable
+# controlhost_terraform_project_name: sample-project
```
Comment (tobias-richter) [inline defaults/main.yml]: same comment as in readme.

File: tasks/env/setup.yml [hunk 3]
```diff
@@ -21,3 +21,26 @@
...
+    - name: "env : setup : Set up Bash aliases"
+      blockinfile:
+        path: "{{ controlhost_bashrc }}"
+        marker: "# {mark} Terraform aliases ANSIBLE MANAGED BLOCK"
+        block: |
+          alias tfp="terraform plan -out plans/plan"
+          alias tfa="terraform apply plans/plan"
+          alias tfi="terraform init"
+
+    - name: "env : setup : Set terraform_project_dir fact"
```
Comment (tobias-richter) [inline tasks/env/setup.yml]: this should not be necessary when we are setting the tf_data_dir directly via a variable.

**Review by tobias-richter — CHANGES_REQUESTED** [review]
Good implementation but we should get rid of the "project" wording. We should use `controlhost_terraform_data_dir` instead.

File: README.md (hunk 1 above)
Comment (bellackn) [reply README.md]: ✔️

File: defaults/main.yml (hunk 2 above)
Comment (bellackn) [reply defaults/main.yml]: ✔️

File: tasks/env/setup.yml (hunk 3 above)
Comment (bellackn) [reply tasks/env/setup.yml]: ✔️

**Pull request closed without merging**